internal/log/level.go

```diff
@@ -98,13 +98,13 @@ func ParseLevel(s string) (level Level, err error) {
 		return Trace, nil
 	case Debug.String(), "DEBUG":
```

Comment:
```suggestion
	case Debug.String():
```

Reply:
added these back with changing padding for CRITICAL